src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs

```diff
@@ -372,6 +372,9 @@ private static bool IsChannel(Type type, out Type payloadType)
 
         private static bool IsStreamed(Type resultType)
         {
+            // TODO: cache reflection for performance, on HubMethodDescriptor maybe?
```

Comment:
File issues for these now. Maybe we can cache at discovery time.

Reply:
He already filed #1546 before submitting the PR :)